cobalt/build/modular_executable.gni

```diff
@@ -17,13 +17,15 @@ template("loader_final_target") {
   if (current_toolchain == default_toolchain) {
     group("${original_target_name}_loader") {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY)
-      deps = [ ":${original_target_name}_copy" ]
+      data_deps = [ ":${original_target_name}_copy" ]
+      write_runtime_deps = "$root_build_dir/${target_name}.runtime_deps"
```

Comment:
Maybe explain why we're writing runtime deps here? Something about how the lib and the loader are two components of one "runnable"?

Reply:
Will do